measurements/api/measurements.py

```diff
@@ -605,10 +605,13 @@ def coal(colname):
             literal_column(f"fp.{colname}"), literal_column(f"mr.{colname}")
         ).label(colname)
 
+    # Merge data from mr_table and fastpath.
+    # Most of the time we prefer data from fastpath, using coal().
+    # For measurement_id, we prefer mr_table. See test_list_measurements_shared
     merger = [
         coal("test_start_time"),
         coal("measurement_start_time"),
-        coal("measurement_id"),
+        func.coalesce(literal_column("mr.measurement_id"), literal_column("fp.measurement_id")).label("measurement_id"),
```

Comment:
This makes sense. Maybe it's worth mentioning in the docstring that `mr.measurement_id` and `fp.measurement_id` have a different prefix.